### PR TITLE
Debugging dartino apps

### PR DESCRIPTION
**Pull request by danrubel**
Various Dartino debugging improvements (see commits listed below).
@devoncarew

### CODE REVIEW

File: lib/dartino/launch_dartino.dart [hunk 1]
```diff
@@ -133,34 +142,45 @@ class DartinoLaunch extends Launch {
...
         .connect(this, 'localhost', observatoryPort)
-        .then((_) {
+        .then((DebugConnection debugger) {
       servicePort.value = observatoryPort;
+
+      // Listen for the first isolate and start the app
+      StreamSubscription<List<DebugIsolate>> subscription;
+      subscription =
+          debugger.isolates.observeMutation((List<DebugIsolate> isolates) {
+        if (isolates.length > 0) {
```
Comment (devoncarew) [inline lib/dartino/launch_dartino.dart]: isolates.isNotEmpty instead? I tend to prefer the more semantic methods when they make sense; YMMV

**Comment by devoncarew** [comment]
lgtm

File: lib/dartino/launch_dartino.dart (hunk 1 above)
Comment (danrubel) [inline lib/dartino/launch_dartino.dart]: Good idea. Will do in my next CL.

**Comment by danrubel** [comment]
Merged https://github.com/dart-atom/dartlang/commit/1502c8d241272a3a8b186c0fbbf307bc740c99b8

**Pull request closed without merging**